### PR TITLE
step 1.17.0-next -> 1.18.0

### PR DESCRIPTION
**Pull request by AlvaroVega**
Mimics https://github.com/telefonicaid/fiware-pep-steelskin/pull/537

### CODE REVIEW

File: Changelog [hunk 1]
```diff
@@ -1,209 +1,218 @@
-1.17.0 (June 12th, 2024)
```
Comment (fgalan) [inline Changelog:1]: Too much change in this file... Maybe due to encoding changes.

Comment (AlvaroVega) [reply Changelog:1]: Just encoding from windows to unix

File: Changelog [hunk 2]
```diff
@@ -1,209 +1,218 @@
...
-
-0.3.0 (December 2nd, 2014)
-
-- Add: Reuse the token instead of authenticating for each request (Issue #22).
-- Add: Support for Keystone as the authentication mechanism.
-- Add: Admin role bypass for privileged usage of the proxy.
-- Add: Plugin to sucure Keypass PAP.
-- Add: Plugin to secure Perseo CEP Rules API.
-- Add: Change expected headers form UUIDs to Names (and resolve UUIDs against Keystone).
+1.18.0 (October 17th, 2024)
```
Comment (fgalan) [inline Changelog:1]:
```suggestion
1.18.0 (October 18th, 2024)
```

File: Changelog (hunk 1 above)
Comment (fgalan) [reply Changelog:1]: Ok. NTC

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**